config/development.yaml

```diff
@@ -13,11 +13,13 @@ persistence:
       cassandra:
         hosts: "127.0.0.1"
         keyspace: "temporal_visibility"
-
-ringpop:
-  name: cadence
-  maxJoinDuration: 30s
-  broadcastAddress: "127.0.0.1"
+server:
+  ringpop:
+    name: cadence
```

Comment:
```suggestion
    name: temporal
```

Reply:
Let follow up with renames in separate PR